lib/modules/manager/mise/index.ts

```diff
@@ -12,5 +26,29 @@ export const defaultConfig = {
   ],
 };
 
-// Re-use the asdf datasources, as mise and asdf support the same plugins.
-export const supportedDatasources = asdfSupportedDatasources;
+const backendDatasources = {
+  core: [
+    GithubReleasesDatasource.id,
+    GithubTagsDatasource.id,
+    JavaVersionDatasource.id,
+    NodeVersionDatasource.id,
+    RubyVersionDatasource.id,
+  ],
+  // Re-use the asdf datasources, as mise and asdf support the same plugins.
+  asdf: asdfSupportedDatasources,
+  aqua: [GithubTagsDatasource.id],
+  cargo: [CrateDatasource.id, GitTagsDatasource.id, GitRefsDatasource.id],
+  dotnet: [NugetDatasource.id],
+  gem: [RubygemsDatasource.id],
+  go: [GoDatasource.id],
+  npm: [NpmDatasource.id],
+  pipx: [PypiDatasource.id, GithubTagsDatasource.id, GitRefsDatasource.id],
+  spm: [GithubReleasesDatasource.id],
+  ubi: [GithubReleasesDatasource.id],
+  // not supported
+  vfox: [],
+};
+
+export const supportedDatasources = deduplicateArray(
+  Object.values(backendDatasources).flat(),
+);
```

Comment:
```suggestion
).sort();
```

so they are sorted alphabetically and not so random